### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by faheem102**

### CODE REVIEW

File: javascript/javascript1/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    let eventDay = daysOfWeek[day.getDay()];
+    return eventDay;
+
+}
+console.log(getEventDay(5));
+
+//Weather Wear
+
+function clothesToWear (temperature) {
+    let myClothes = "t-shirt and jeans";
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:35]: const

File: javascript/javascript1/week2/index.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+//Student Manager
+
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+
+  // You write code here
+
+    if (class07Students.indexOf(studentName) != -1 &&
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:59]: nice solution :+1:

File: javascript/javascript2/week2/index2.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
+function getFullname (firstname, surname, useFormalname, gender){
```
Comment (aina21) [inline javascript/javascript2/week2/index2.js:1]: remove duplicate file

File: javascript/javascript2/week2/index2.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (aina21) [inline javascript/javascript2/week2/index2.html:1]: remove duplicate file

**Review by aina21 — CHANGES_REQUESTED** [review]
nice job :+1:

File: javascript/javascript2/week2/index2.html (hunk 4 above)
Comment (faheem102) [reply javascript/javascript2/week2/index2.html]: hi aina thanks for you feedback i have now implemented suggested changes. please have a look if i have done it in right way. thank you

**Review by aina21 — APPROVED** [review]

**Pull request merged**